showcase/app/templates/components/icon.hbs

```diff
@@ -0,0 +1,135 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+{{page-title "Icon"}}
+
+<Shw::Text::H1>Icon</Shw::Text::H1>
+
+<section data-test-percy>
+
+  <Shw::Text::H2>Size</Shw::Text::H2>
+
+  <Shw::Text::Body>Base sizes</Shw::Text::Body>
+
+  <Shw::Flex as |SF|>
+    <SF.Item @label="16px (default)">
+      <Hds::Icon @name="bug" />
+    </SF.Item>
+    <SF.Item @label="24px">
+      <Hds::Icon @name="bug" @size="24" />
+    </SF.Item>
+  </Shw::Flex>
+
+  <Shw::Text::Body>Custom sizes</Shw::Text::Body>
+
+  <Shw::Flex as |SF|>
+    <SF.Item @label="12px (stretched)">
+      <div {{style width="12px" height="12px"}}>
+        <Hds::Icon @name="bug" @size="16" @stretched={{true}} />
+      </div>
+    </SF.Item>
+    <SF.Item @label="32px (stretched)">
+      <div {{style width="32px" height="32px"}}>
+        <Hds::Icon @name="bug" @size="24" @stretched={{true}} />
+      </div>
+    </SF.Item>
```

Comment:
suggestion
in this way we test also the combination `stretched` + `isInline` and make sure it still works as expected
```suggestion
    </SF.Item>
    <SF.Item @label="32px (stretched + inline)">
      <div {{style width="32px" height="32px"}}>
        <Hds::Icon @name="bug" @size="24" @stretched={{true}} @isInline={{true}} />
      </div>
    </SF.Item>
```